### ISSUE TITLE
Multiple else cases don't work in Html

### ISSUE THREAD

**Issue by boostermedia** [boostermedia/clopp#6]
For example:
```html
    <!-- #if __chrome
    <script id="booster-api" type="text/javascript" src="../../shared/booster/api.js"></script>
    <!-- #elseif __target='dev'  -->
    <script id="booster-api" type="text/javascript" src="shared/booster/api.js"></script>
    <!-- #endelseif
    <!-- #elseif __target='prod' || __target='stage'
    <script id="booster-api" type="text/javascript" src="/shared/booster/api.js"></script>
    <!-- #endeif -->
```

**Comment by RikoOphorst** [comment]
You made a typo in  your closing if statement:



Should be:

**Comment by RikoOphorst** [comment]
I mean:

#endeif

Should be

#endid

**Comment by RikoOphorst** [comment]
Goddammit. Stupid phones.

#endif

**Comment by bnRietveld** [comment]
Spam :D

**Comment by boostermedia** [comment]
My bad, but the bug is still there.
In the Clopp test.html, the following doesn't work as expected.
```
    <!-- #if __environment=='prod'
        <title>Production</title>
    <!-- #elseif __environment=='stage' -->
        <title>Stage</title>
    <!-- #else
        <title>undefined</title>
    <!-- #endif -->
```
I'm looking into it though :)

**Comment by boostermedia** [comment]
Fixed in 0.2.1 (had to do with it's parsing of strings)
